ush/forecast_predet.sh

```diff
@@ -134,6 +134,29 @@ common_predet(){
   # Several model components share DATA/INPUT for input data
   if [[ ! -d "${DATA}/INPUT" ]]; then mkdir -p "${DATA}/INPUT"; fi
 
+  # For CMEPS, CICE, MOM6 and WW3 determine restart writes
+  # Note FV3 has its own restart intervals  
+  cmeps_restart_interval=${restart_interval:-${FHMAX}}
```

Comment:
This isn't a common variable if it starts w/ `cmeps`.

Reply:
What would you prefer it be called?  This is for CMEPS, WW3, ICE, and MOM6

Reply:
It is not the name, but the place these are introduced.
The function `CMEPS_predet` IMO, would be appropriate to introduce these variables related to `cmeps_` (and more general the coupled model since they share these concepts)

Reply:
Thanks for the additional explanation.  I will move it to CMEPS!